enh(release): Add Centreon Web 22.04.5 RN (#1817)
centreon/centreon-documentation: versioned_docs/version-22.04/releases/centreon-core.md

## Centreon Web

### 22.04.5

#### Enhancements

> - [UX] Improved usability to access logs and reports button in details panel

Reviewer: ```suggestion
- [UX] Improved usability when accessing logs and reports button in details panel
```

#### Bug fixes

- [Administration] Fixed selection of options in second select box in ACL Group configuration page
- [Authentication] Fixed contact_location when creating a session from OpenId Connect which made it impossible to access MBI jobs

> - [Configuration] Fixed an error in the Configuration > Services > Templates menu causing display of HTML code

Reviewer: ```suggestion
- [Configuration] Fixed an error in the Configuration > Services > Templates menu causing HTML code to be displayed
```

> - [Configuration] Fixed duplication of a Remote Server

Reviewer: ```suggestion
- [Configuration] Fixed error that occurred when duplicating a Remote Server
```

- [Core] Correct escapeSecure usage
- [Core] Fixed database partitioning for Aurora DBMS
- [Install] Fixed Centreon Web installation using sources and silent mode
- [Install] Fixed default PHP-FPM configuration on Debian

> - [Resources Status] Fixed the "Alias" tile that was not displayed in detail panel for 22.04

Reviewer: ```suggestion
- [Resources Status] Fixed the "Alias" tile that was not displayed in details panel
```

> - [Resources Status] Fixed the broken link to host right panel

Reviewer: ```suggestion
- [Resources Status] Fixed broken link to host right panel
```

> - [UI] Fixed HTML code appearing instead of svg icon with Firefox

Reviewer: ```suggestion
- [UI] Fixed HTML code appearing instead of SVG icon with Firefox
```

- [UI] Fixed french typo in OpenID Connect configuration form
- [UI] Fixed untranslated label in Resources Status

> - [Widgets] Retrieve possibility to not select poller in preferences

Reviewer: ```suggestion
- [Widgets] Restored possibility to not select a poller in preferences
```

#### Security fixes

- [CLAPI] Added a check to verify that the user has the admin role
- [CLAPI] Sanitized and bound CLAPI poller configuration queries
- [Configuration] Fixed SQLi in poller's resource creation
- [Configuration] Sanitized and bound Meta Service configuration queries
- [Configuration] Sanitized and bound command configuration queries
- [Configuration] Sanitized and bound graph configuration queries
- [Configuration] Sanitized and bound queries in contactgroup file
- [Configuration] Sanitized and bound queries in listServiceCategories file
- [Configuration] Sanitized and bound queries in listVirtualMetrics file
- [Configuration] Sanitized and bound queries in service argumentsXml file
- [Configuration] Sanitized and bound queries in service host categories file
- [Configuration] Sanitized and bound queries in servicegroup_dependency file
- [Configuration] Sanitized and bound templates of service listing queries

> - [Monitoring] Fixed XSS vulnerability reflected in deprecated services status details page

Reviewer: ```suggestion
- [Monitoring] Fixed XSS vulnerability in deprecated services status details page
```

## Centreon Web

### 22.04.5

> 

Reviewer: ```suggestion

Release date: `September 20, 2022`

```

#### Enhancements

- [UX] Improved usability when accessing logs and reports button in details panel

#### Bug fixes

- [Administration] Fixed selection of options in second select box in ACL Group configuration page
- [Authentication] Fixed contact_location when creating a session from OpenId Connect which made it impossible to access MBI jobs
- [Configuration] Fixed an error in the Configuration > Services > Templates menu causing HTML code to be displayed
- [Configuration] Fixed error that occurred when duplicating a Remote Server

> - [Core] Clean code in forMyAccount

Reviewer: ```suggestion
- [Core] Cleaned code in forMyAccount
```

> - [Core] Correct escapeSecure usage

Reviewer: ```suggestion
- [Core] Corrected escapeSecure usage
```
